src/Mondu/Mondu/Presenters/PaymentInfo.php

```diff
@@ -88,10 +89,19 @@ public function get_mondu_section_html() {
                 <span><strong><?php printf(__('Mondu id:', 'mondu')); ?></strong></span>
                 <span><?php printf($order_data['uuid']); ?></span>
             </p>
+            <?php
+                if(!$invoice_data) {
+                    ?>
+                        <button <?php $order_data['state'] === 'canceled' ? printf('disabled') : ''?> data-mondu='<?php echo(json_encode(['order_id' => $this->order->get_id()])) ?>' id="mondu-create-invoice-button" type="submit" class="button grant_access">
```

Comment:
I think we should also check if the invoice is not also already created (state different than shipped?).

Reply:
We can check the order state to be confirmed or partially_shipped instead of checking for invoices.
What do you think?